docs/aerodromes/classc/Amberley.md

```diff
@@ -26,44 +26,82 @@ AMB ADC owns the Class C airspace **in the AMB CTR** within 10nm of the YAMB ARP
 
 !!! phraseology
-    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.
-    **AMB ADC**: "CRNG21, cleared to YAMB via MOSSI, flight planned route. Runway 15, Northern 1 Departure. Climb to `F190`. Squawk 3601, Departure Frequency 126.20"  
+    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.  
+    **AMB ADC**: "CRNG21, cleared YAMB via MOSSI, flight planned route. Runway 15, Northern 3 Departure. Climb `F190`. Squawk 3601, Departure Frequency 126.20"   
+
+Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
 
 !!! phraseology
-    WOLF03 was assiged the BYRON7 departure with their initial clearance. 
+    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
     **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
     **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
 
 | Departure | Initial Constraint | Route |
 | --------- | ----------| --------- |
-| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
-| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
-| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
-
-!!! note
-    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
+| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |
 
 !!! note
     Since the Coded Departures are not SIDs, AMB ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.
 
+!!! abstract "Reference"
+    Charts for the Coded Departures are available in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
 
 ### Runway Operations
 Runways 15/33 are the primary runways at YAMB. 
 
-## Charts
+## Helicopter Operations
+
+### Taxiway Alpha
+
+Taxiway Alpha has been divided into portions for helicopter operations as follows;
+
+| Location | Name |
+| ------ | ----------|
+| Between A2 and A4     | Alpha North  |
+| Between A2 and RWY 04/22     | Alpha Centre |
+| Between A1 and RWY 04/22     | Alpha South |
+| TWY Q and A Junction     | Holding Point Quebec |
+
+<figure markdown>
+![YAMB Helicopter Taxiway Alpha](img/yamb_heli_taxiwayalpha.png){ width="600" }
+  <figcaption>YAMB Helicopter Taxiway Alpha</figcaption>
+</figure>
```

Comment:
Are we able to get a phraseology example of how an instruction may be issued to operate in these areas? And is this for arrivals and departures, or operations in ground effect?

Reply:
Just operations in ground effect, eg: practising winching operations. But i'd imagine that it can also be nominated as a HLS, however about 90% of the time aircraft would be assumed to land in the Taxiway W junction.

Reply:
Yeah a bit of guidance on what these areas are used for would be great.

I'd imagine it would be fairly close to that, likely 'not above 100ft' perhaps. I'd imagine winching ops would sometimes involve out of ground effect hovering, but it would be up to the pilot to request their particular operations. If we can at least get an indication of what the areas are used for and an example, that will give controllers 90% of the answer, they can work the last 10% out themselves.

Reply:
There isn't really any public knowledge to which areas are used for which type of operations, but i'll most definitely add an example of phraseology for one of the local helicopter callsigns and requesting operations for certain areas.

Reply:
Yeah easy. When you say that 90% of the time aircraft would be presumed to land in the taxiway Q junction, do you mean inbound aircraft or aircraft who have been operating in those heli areas and are now done?

Reply:
Both, typically the local helicopter CHOP41 will land on the taxiway Q junction after operations in one of the areas, circuits, or a VFR departure off to some random SAR anchor. This is as all their staff and equipment are based in that location.

Reply:
Ok great, perhaps we could say something along the lines of 'helicopters may be processed as per fixed wing ops or from Holding Point Quebec at pilot request'?

Reply:
Yeah that sounds good

Reply:
I've added an example phraseology for clearing an aircraft doing operations within the taxiway alpha areas, minus the squawk as in real life CHOP41 isn't verbally issued one (as per AD2 SUP), and additionally I assumed it wouldn't be required considering the helicopter wouldn't be leaving the circuit area and require identification. 

Haven't put anything in yet regarding > 'helicopters may be processed as per fixed wing ops or from Holding Point Quebec at pilot request' feel free to add it where you feel it suits best.

Reply:
No worries, I'll have a quick squizz